pkg/services/ngalert/api/api_alertmanager_silences.go

```diff
@@ -29,7 +31,22 @@ func (srv AlertmanagerSrv) RouteGetSilence(c *contextmodel.ReqContext, silenceID
 	if err != nil {
 		return response.ErrOrFallback(http.StatusInternalServerError, "failed to get silence", err)
 	}
-	return response.JSON(http.StatusOK, SilenceToGettableSilence(*silence))
+
+	silenceWithMetadata := &models.SilenceWithMetadata{
+		Silence: silence,
+	}
+	if c.QueryBool("accesscontrol") {
```

Comment:
Also, I wonder if it could be enough to just check existence of parameter, for example `/silences?accesscontrol&metdata`?

Reply:
for `accesscontrol` I'm following the same style as the existing query params for other resources (ex: [Folders](https://github.com/grafana/grafana/blob/1208888bb618774228aefe65884e06316a06e714/pkg/api/folder.go#L432-L434))